src/test/ui/issues/issue-76547.rs

```diff
@@ -0,0 +1,38 @@
+// Test for for diagnostic improvement issue #76547
```

Comment:
nit: you have the word "for" twice :)

```suggestion
// Test for diagnostic improvement issue #76547
```